datatypes/src/error.rs

```diff
@@ -5,8 +5,9 @@ use crate::{
     spatial_reference::SpatialReference,
 };
 use snafu::{prelude::*, AsErrorSource, ErrorCompat, IntoError};
-use std::{any::Any, convert::Infallible, sync::Arc};
+use std::{any::Any, convert::Infallible, path::PathBuf, sync::Arc};
 
+pub type Result<T, E = Error> = std::result::Result<T, E>;
```

Comment:
this is already defined in utils, just use that one.

Reply:
Had to change the return type of `canonicalize_subpath` as well, but that was the actual reason for declaring the other type, so I guess thats ok :)
https://github.com/geo-engine/geoengine/pull/639/commits/2601f19b1351e51893cd0a90d02f9e2e12e4990a